node-scripts/src/pipeline3/impl/dockerMonolith.js

```diff
@@ -0,0 +1,372 @@
+"use strict";
+
+const _ = require("lodash");
+
+const cst = require("../../const");
+const heredoc_2 = cst.HEREDOC_2;
+const utils = require("../../utils/utils");
+/**
+ * Implementation of script utils to specifically manipulate Docker containers.
```

Comment:
Scripts to manipulate Docker "Monolith" type of containers.

Reply:
Done